### PR TITLE
Fix/lp 191 fix community wall

### PR DESCRIPTION
**Pull request by irv205**
Before:

image

After:

image

### CODE REVIEW

File: feature/communitywall/src/main/java/com/iteneum/communitywall/presentation/CommunityWallView.kt [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+import androidx.compose.foundation.layout.*
 import androidx.compose.foundation.lazy.LazyColumn
 import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import com.iteneum.communitywall.R
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
```
Comment (melissaquezada94) [inline feature/communitywall/src/main/java/com/iteneum/communitywall/presentation/CommunityWallView.kt]: Please delete the unused imports

File: feature/communitywall/src/main/java/com/iteneum/communitywall/presentation/CommunityWallView.kt [hunk 2]
```diff
@@ -70,7 +76,7 @@ fun CommunityWallItemList() {
     ){
         LazyColumn{
             items(postList){
-                LpPostCard(modifier = Modifier, it.name, it.img, it.time, it.message, {}, {})
+                LpPostCard(modifier = Modifier.padding(size.smallerSize), it.name, it.img, it.time, it.message, {}, {})
```
Comment (carlosph816) [inline feature/communitywall/src/main/java/com/iteneum/communitywall/presentation/CommunityWallView.kt]: Can we reformat code please?

Comment (irv205) [reply feature/communitywall/src/main/java/com/iteneum/communitywall/presentation/CommunityWallView.kt]: done

File: feature/communitywall/src/main/java/com/iteneum/communitywall/presentation/CommunityWallView.kt (hunk 1 above)
Comment (irv205) [reply feature/communitywall/src/main/java/com/iteneum/communitywall/presentation/CommunityWallView.kt]: done

**Review by melissaquezada94 — APPROVED** [review]
well done!

**Pull request merged**